dreem/models/embedding.py

```
@@ -136,27 +161,92 @@ def _check_init_args(self, emb_type: str, mode: str):
                 f"Embedding `mode` must be one of {self.EMB_MODES} not {mode}"
             )
 
-    def forward(self, seq_positions: torch.Tensor) -> torch.Tensor:
+        if mode.lower() == "rope" and self.embedding_agg_method == "average":
+            raise ValueError(
+                f"Cannot use aggregation method 'average' for rope embedding; must use 'stack' or 'concatenate'"
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)